hire/hire/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for hire project.
+
+Generated by 'django-admin startproject' using Django 3.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'whw@5v^e^ub!93z$b5)3s!ees$k*anv63z2zyjcabjec8hc4f%'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Do we want to remove this in this file?

Reply:
I think we meant to say "Do we want to remove this variable `DEBUG`". That's a very good question. We want to ensure that we do not have DEBUG set to True in production.

However, we need it to be true in development. It will make the development very easy. Let's make a note (or even a GitHub issue) to ensure this is removed before it goes to production. I always ensure that it is off for security reasons.